UI/DevScreens.cpp

```diff
@@ -720,6 +722,18 @@ void SystemInfoScreen::CreateViews() {
 		for (auto &feature : features) {
 			gpuExtensions->Add(new TextView(feature, new LayoutParams(FILL_PARENT, WRAP_CONTENT)))->SetFocusable(true);
 		}
+		
+		// Vulkan specific code here, can't be bothered to abstract.
+		gpuExtensions->Add(new ItemHeader(si->T("Display Color Formats")));
+		VulkanContext *vk = (VulkanContext *)draw->GetNativeObject(Draw::NativeObject::CONTEXT);
```

Comment:
Wait - I don't think we use rtti or anything, this will just blindly cast a d3d11 context or similar to Vulkan.  We'd need to check GetGPUBackend().

-[Unknown]

Reply:
That is checked, just a few lines above.

Reply:
Oh sorry, I expanded quickly but misread the indentation...

-[Unknown]